### ISSUE TITLE
Cannot change tab order

### ISSUE THREAD

**Issue by codemusket** [nico3333fr/van11y-accessible-modal-window-aria#7]
Hello, I have tested a fix to my problem, and am wondering if it can be implemented. I need to be able to tab to a button in my content first before the X button. Currently the X button takes taborder precedence because it is implemented first in the modal creation. By simply creating the X button html after the content, this can be done. This could be a customized feature if you believe X button should be first in some cases.

Screen readers are reading the x button first, which is not the primary action to be taken, only secondary.

**Comment by nico3333fr** [comment]
Fixed with #6
